script/Ciel/Day 5/05_01_CIEL05_3E.txt

```diff
@@ -66,17 +66,17 @@ She simply isn't human, and operates on a fundamentally different set of values
 [sha:f01c3f8137ced2211ee2cdf83ec751f52b8c9934]{
 -- Page 580, Offset 12366.
 -- だから―――あいつに人間の常識なんてものを期待した、俺の方が馬鹿だった。
-%{i}I'm%{/i} the fool here―――for expecting her to act the same way a human would.
+%{i}I'm%{/i} the fool here――for expecting her to act the same way a human would.
```

Comment:
Might not be needed? seems like a JP artifact

Reply:
Mm I think this one's good since it hammers the connection to the line above
> It's not like she's foolish or evil.
> ...
> %{i}I'm%{/i} the fool here――for

Reply:
Okay, that's fine.